### DISCUSSION TITLE
Depth Anything Version 2

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#130] [General]
Hi, there is a new 3d model called "Depth Anything Version 2" it's faster and supposedly more accurate than the original depth anything. Would you consider adding it to iw3.  Here is depth anything 2 on this site https://huggingface.co/depth-anything/Depth-Anything-V2-Large

**Comment by nagadomi** [comment]
I will check it later.
At this time, it seems that DepthAnything's Github organization has been taken private.
And the model of that link is `License: cc-by-nc-4.0`. Only the small model is `License: apache-2.0`.

**Reply by loawizard** [reply]
oew that does not sound good that it's been taken private. What is the implications of that?

**Reply by loawizard** [reply]
hmmn Does'nt the license just mean you can't use it to make money. Well that's not a problem with your program as you are not asking payment for it. You might get donations to you as a developer but your not selling the software.

**Comment by loawizard** [comment]
Wow Very cool!

**Comment by loawizard** [comment]
wow this youtube video about depth anything V2 is making me excited ! https://www.youtube.com/watch?v=zna5PrtDUFE

Apparently their gethub is down but they transfered all the files to  huggingface for now https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/blob/main/README_Github.md

**Comment by nagadomi** [comment]
Supported. For `Any_V2_B` and `Any_V2_L`, see https://github.com/nagadomi/nunif/tree/master/iw3#about-any_v2_b-and-any_v2_l

I have supported it on Depth-Anything-V1 source code, without using the official V2 source code. I haven't tested it much. It is possible that there is a problem.

**Reply by loawizard** [reply]
Amazing Work. We will see how it works.

How do I agree to the pre-trained model license and place the checkpoint file Myself for Any V2 B and Any V2 L?
Thank you so much
....

Edit: Figured it out, you have to download the files from  https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/blob/main/README_Github.md
and put in in the \nunif\iw3\pretrained_models\hub\checkpoints folder

**Comment by diverswan9** [comment]
Just a quick review... I haven't tried a great variety of types of scenes but from what I've seen so far... I used 0.8 convergence and 2.2 divengence. BaseV2 seems to want to lean the subjects back rather than upright. Overall V2 brings out more detail. Close ups of faces/clothing have more sculpting but do lean back. Overall the BaseV1 seems better at separating layers. Both took about  the same amount of time to process. Thin wires/sticks/ropes come out better with V2 but it's a trade off with V1's better layers representation. This is what it looks like so far. I'll  keep testing further....  CHEERS!

**Reply by carlobs** [reply]
Same here.... i prefer BaseV1 at the moment.
